man/kernel-install.xml

```diff
@@ -172,6 +170,30 @@
     either 32 hexadecimal characters or the special value <literal>Default</literal>.</para>
     <para><varname>KERNEL_INSTALL_BOOT_ROOT=</varname> is set for the plugins to the root directory (mount point, usually) of the hierarchy
     where boot-loader entries, kernel images, and associated resources should be placed. Can be overridden by setting <varname>BOOT_ROOT=</varname>.</para>
+
+    <para><varname>KERNEL_INSTALL_LAYOUT=bls|other...</varname>
+    is set for the plugins to select the target layout hierarchy – free-form.
+    Defaults to "bls" if <filename>$BOOT/<replaceable>MACHINE-ID</replaceable></filename> exists, or "other" otherwise;
+    overriden by setting <varname>layout=</varname> in <filename>install.conf</filename>,
```

Comment:
```
<varname>KERNEL_INSTALL_LAYOUT=bls|other|...</varname> specifies the installation layout.
Defaults to <option>bls<option> if <filename>$BOOT/<replaceable>MACHINE-ID</replaceable></filename> exists, or "other" otherwise.
Additional layout names may be defined by convention. If a plugin uses a special layout, it is encouraged to declare it's own layout name and configure <varname>layout=</varname> in <filename>install.conf</filename> upon initial installation.
```